src/avram/attribute.cr

```diff
@@ -59,11 +66,13 @@ class Avram::Attribute(T)
     errors.empty?
   end
 
-  def value=(@value)
-    @changed = true
+  def changed?(**arguments)
+    from = arguments[:from]? ? arguments[:from]? == @original_value : true
+    to = arguments[:to]? ? arguments[:to]? == @value : true
+    @value != @original_value && to && from
   end
 
-  def changed?
-    @changed
+  def changes
```

Comment:
I think that instead of returning an array of changes it would be cool to add a getter for “original_value” that way you can call the methods for what you want and not worry about arrays. Thoughts?

Reply:
There is already a getter for `original_value`:

```cr
def original_value
  ensure_no_blank(@original_value)
end
```

So, we might as well ditch the whole `changes` method. It already felt superfluous. I probably tried to match Rails' behaviour too much here.

Reply:
Sounds good to me!